AuxTel/AuxTel-Non-standard-Operations/AuxTel-Emergency-Shutdown/AuxTel-Emergency-Shutdown.rst

```diff
@@ -0,0 +1,114 @@
+.. _Weather constraints: https://obs-ops.lsst.io/Observing-Constraints/AuxTel-Observing-Constraints/AuxTel-Weather-Constraints-index.html
+.. Review the README in this procedure's directory on instructions to contribute.
+.. Static objects, such as figures, should be stored in the _static directory. Review the _static/README in this procedure's directory on instructions to contribute.
+.. Do not remove the comments that describe each section. They are included to provide guidance to contributors.
+.. Do not remove other content provided in the templates, such as a section. Instead, comment out the content and include comments to explain the situation. For example:
+    - If a section within the template is not needed, comment out the section title and label reference. Include a comment explaining why this is not required.
+    - If a file cannot include a title (surrounded by ampersands (#)), comment out the title from the template and include a comment explaining why this is implemented (in addition to applying the ``title`` directive).
+.. Include one Primary Author and list of Contributors (comma separated) between the asterisks (*):
+.. |author| replace:: *Karla Peña Ramírez*
+.. If there are no contributors, write "none" between the asterisks. Do not remove the substitution.
+.. |contributors| replace:: *Craig Lage, Eric Christensen, OS team*
+.. This is the label that can be used as for cross referencing this procedure.
+.. Recommended format is "Directory Name"-"Title Name"  -- Spaces should be replaced by hyphens.
+.. _AuxTel-Non-Standard-Operations-AuxTel-Emergency-Shutdown: 
+.. Each section should includes a label for cross referencing to a given area.
```

Comment:
Not sure why this line is not colored as a comment.

Reply:
It looks colored in mine.